compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageContext.java

```diff
@@ -306,4 +307,14 @@ PackageContext duplicate(Project project) {
                 this.cloudTomlContext, this.compilerPluginTomlContext, this.packageMdContext,
                 this.compilationOptions, duplicatedModuleContextMap, this.pkgDescDependencyGraph);
     }
+
+    public Set<ModuleLoadRequest> getAllImports() {
```

Comment:
Lets keep this package private. To make it accessible in the CLI module, expose it as a util via the ProjectUtils